packages/fe/modules/search/plugins/index.js

```diff
@@ -29,4 +29,13 @@ export default async function ({ app, store }, inject) {
     app.$search.clearSearchQuery()
     app.$filter.clearAll()
   })
+  inject('clearAllFilters', (exception) => {
+    app.$filter.clearAll(exception)
+  })
+  inject('clearAll', () => {
+    app.$search.clearSearchQuery()
+    app.$filter.clearAll()
+    app.$sort.clearQuery()
+    app.$limit.clearQuery()
+  })
```

Comment:
Let's rename this to something more specific since this becomes a global helper!

Reply:
done